### DISCUSSION TITLE
Migration from azure to GitHub (lfs fail)

### DISCUSSION THREAD

**Discussion by HariReddy10** [community/community#40009] [Other Feature Feedback, Questions, & Ideas]
**Commands what I'm executing to migrate azure to GitHub** 
![IMG_20221125_003849](https://user-images.githubusercontent.com/39373680/203852170-fdcfe410-97df-41c8-b425-09c987f7a7f3.jpg)
**Error I'm facing** 
![IMG_20221125_003545](https://user-images.githubusercontent.com/39373680/203852180-e80cd43c-7f03-4fa9-988d-9511b4a263f5.jpg)
### Select Topic Area

Question

### Body

When I'm trying to migrate azure to GitHub getting an error limit is 25mb , but my war and jar file have size 80mb cloud you please me on migration from azure to GitHub.

**Comment by ossanna16** [comment]
Hi there @HariReddy10 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by airtower-luna** [comment]
Installing LFS is not enough, you'll have to actually remove the large files from the repository history and put them on LFS instead. See [Removing files from a repository's history](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#removing-files-from-a-repositorys-history) for how to do that.

> but my war and jar file have size 80mb

However, WAR and JAR files usually shouldn't be in a Git repository at all, and instead be either built from source, or when they are dependencies downloaded from a suitable package repository (e.g. using Maven, depending on your build system).
